support-frontend/assets/helpers/storage/abandonedBasketCookies.ts

```diff
@@ -19,13 +39,8 @@ export function useAbandonedBasketCookie(
 		region,
 	};
 
-	// support-dotcom-components can only return a user to the checkout for these products
-	// don't drop the cookie if they came from a different checkout
-	const isSupportedProduct =
-		product === 'Contribution' || product === 'SupporterPlus';
-
```

Comment:
I've removed this function but wondering whether are any reasons not too... @tomrf1? we could just add in an or statement to include one time contribution as well

Reply:
So `'Contribution'` is for recurring contributions?
It depends if `useAbandonedBasketCookie` is ever called from a checkout that isn't supported, I don't know the answer to this!

Reply:
Yes `Contribution` is for recurring contributions.

I've re-added this with OneTimeContribution instead as it's such a small addition!